sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Orders.java

```diff
@@ -0,0 +1,40 @@
+package com.exadel.sandbox.team5.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+
+@Entity
+@Table(name = "orders")
```

Comment:
Entity represents single record in DB, so should be non plural. Same for table name

Reply:
I can't use word obred, because it's keyword is SQL. I renamed orders to booking.

Reply:
did you try to change: "orders" -> "`order`"?

Reply:
Yes, I did. That is why I named it "orders".
I can't create table with name "order"(when I try it in MySql Workbench I have got a error).